network/transparent_vlan_endpointclient_linux.go

```diff
@@ -673,6 +675,17 @@ func (client *TransparentVlanEndpointClient) DeleteEndpointsImpl(ep *endpoint, _
 	return nil
 }
 
+// Creates a new retrier with a fixed delay, and treats all errors as retriable
+func (client *TransparentVlanEndpointClient) Retry(f func() error) error {
+	retrier := retry.Retrier{
+		Cooldown: retry.Max(numRetries, retry.Fixed(sleepDelay)),
+	}
+	return errors.Wrap(retrier.Do(context.Background(), func() error {
```

Comment:
Are there no timeouts and such defined on the client that could be bound to a context?